WordPress/build.gradle

```diff
@@ -426,9 +426,12 @@ dependencies {
 
     implementation ('com.github.indexos.media-for-mobile:android:43a9026f0973a2f0a74fa813132f6a16f7499c3a')
 
-    implementation "$gradle.ext.wputilsBinaryPath:$wordPressUtilsVersion"
-    testImplementation "$gradle.ext.wputilsBinaryPath:$wordPressUtilsVersion"
-    debugImplementation "$gradle.ext.wputilsBinaryPath:$wordPressUtilsVersion"
+    implementation ("$gradle.ext.wputilsBinaryPath") {
```

Comment:
All three were identical, so I used only `implementation`. 

I added the strict mode to prevent gradle from wrongly considering a pinned version (eg. 1.22) as more recent than a commit-hash version (eg. trunk-1ed207c03d2242b6fc3d74f9e388e9163cbc82a6).

Reply:
Thanks for updating this. It should have been updated while doing the composite build but at the time I was also worried about missing something 😅